net/softethervpn/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
```

Comment:
Well in all makefiles I have always used the same copyright (obviously it's for the Makefile not for the program) and there is the same heading in every other makefile. Why do I have to remove it ?